fizz-buzz-streams/src/test/java/com/pubsap/eng/streams/FizzBuzzAggregatorTest.java

```diff
@@ -0,0 +1,103 @@
+package com.pubsap.eng.streams;
+
+import com.pubsap.eng.schema.*;
+
+import org.junit.jupiter.api.Test;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+
+public class FizzBuzzAggregatorTest {
+
+    @Test
+    public void aggregatorShouldUpdateFizz() {
```

Comment:
Suggestion: `aggregatorShouldGroupItemBasedOnItemValue`

Also, dont split the cases, this is a single feature (example):
```java
Item givenItem1 = new Item(ItemValue.Fizz);
Item givenItem2 = new Item(ItemValue.Fizz);
Item givenItem3 = new Item(ItemValue.Buzz);
Item givenItem4 = new Item(ItemValue.FizzBuzz);
Item givenItem5 = new Item(ItemValue.None);
Item givenItem6 = new Item(ItemValue.None);
Item givenItem7 = new Item(ItemValue.None);
```
... I have copy pasted things here but Using `Collections` is better of course